README.md

```diff
@@ -242,6 +242,37 @@ Copies (or appends to) a field from an existing field.
 copy_field("<sourceField>", "<targetField>")
 ```
 
+##### `debug_record`
+
+Prints the current record either to standard output or to a file.
```

Comment:
```suggestion
Prints the current record at the position of the function either to standard output or to a file. It can help to debug the  fix-transformation.
```